<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>991</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>881</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(195, 190, 180);
border-radius:30px
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>971</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,210);
</string>
      </property>
      <widget class="QWidget" name="horizontalWidget_3" native="true">
       <property name="geometry">
        <rect>
         <x>990</x>
         <y>0</y>
         <width>81</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:rgba(0,0,0,0);
	color:rgb(255,255,255);
	border-radius:1px;
	font-size:18px;
	font-family:dripicons-v2;
}

QPushButton:hover{
	background-color:rgb(49,48,53)
}

QPushButton#cerrar:hover{
	background-color:rgb(232,17,25)
}

QpushButton:pressed{
	padding-top:5px;
 	padding-left:5px;
}
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>121</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgba(255,255,255,255)</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;PROVEDORES&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalWidget_4" native="true">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>0</y>
         <width>91</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:rgba(0,0,0,0);
	color:rgb(255,255,255);
	border-radius:1px;
	font-size:18px;
	font-family:dripicons-v2;
}

QPushButton:hover{
	background-color:rgb(49,48,53)
}

QPushButton#cerrar:hover{
	background-color:rgb(232,17,25)
}

QpushButton:pressed{
	padding-top:5px;
 	padding-left:5px;
}
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnCerrar">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnInsertar">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>300</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">
border:1.5px solid #7c6e59;
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Insertar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>341</width>
        <height>451</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);
border-radius:30px</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="tblProvedores">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>511</width>
        <height>371</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
border: 3px solid  #000000;
background-color: rgb(199, 194, 184);
</string>
      </property>
      <column>
       <property name="text">
        <string>RUC</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
       <property name="background">
        <color>
         <red>228</red>
         <green>222</green>
         <blue>211</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Raz.Social</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Direccion</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Telefono</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNombre">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
border:1.5px solid #7c6e59;
border-radius:15px;
color: rgb(0, 0, 0);
padding-bottom:4px;
padding-left:4px</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string>Raz. Social:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtRazSocial">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
border:1.5px solid #7c6e59;
border-radius:15px;
color:rgba(0,0,0,240);
padding-bottom:4px;
padding-left:4px</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCancelar">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>340</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">
border:1.5px solid #7c6e59;
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>350</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/newPrefix/Marohuma_icono.png);
background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Imagenes/QT_imagenes.qrc">:/newPrefix/Marohuma_icono.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string>Direccion:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDireccion">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
border:1.5px solid #7c6e59;
border-radius:15px;
color:rgba(0,0,0,240);
padding-bottom:4px;
padding-left:4px</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string>Telefono:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtTelefono">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
border:1.5px solid #7c6e59;
border-radius:15px;
color:rgba(0,0,0,240);
padding-bottom:4px;
padding-left:4px</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string>RUC:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtRuc">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
border:1.5px solid #7c6e59;
border-radius:15px;
color:rgba(0,0,0,240);
padding-bottom:4px;
padding-left:4px</string>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>widget_2</zorder>
     <zorder>tblProvedores</zorder>
     <zorder>label_8</zorder>
     <zorder>txtNombre</zorder>
     <zorder>label_9</zorder>
     <zorder>txtRazSocial</zorder>
     <zorder>label</zorder>
     <zorder>label_11</zorder>
     <zorder>txtDireccion</zorder>
     <zorder>label_12</zorder>
     <zorder>txtTelefono</zorder>
     <zorder>label_13</zorder>
     <zorder>txtRuc</zorder>
     <zorder>btnCancelar</zorder>
     <zorder>btnInsertar</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Imagenes/QT_imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
